content/docs/studio/rest-api.md

```diff
@@ -103,6 +104,148 @@ Running this example will fail because it requires that you have [set up remote
 cloud credentials] in DVC Studio to where the model is stored.
 
 </admon>
+</toggle>
+
+## Create new project
+
+Create a new project programmatically. Authentication is required using a token,
+which can be generated from the user's profile settings in DVC Studio.
+
+```yaml
+Endpoint: api/create-project
+HTTP Method: POST
+Content-Type: application/json
+```
+
+### Authentication
```

Comment:
just to double check - Authorization or Authentication? :)

Reply:
It is "authentication" as we discussed during the implementation of device auth.